src/index.js

```diff
@@ -5,7 +5,11 @@
 import React from 'react'
 import {css, styleSheet} from 'glamor'
 import shouldForwardProperty from './should-forward-property'
-import domElements from './dom-elements'
+
+const htmlTagNames = require('html-tag-names')
+const svgTagNames = require('svg-tag-names')
```

Comment:
Instead of `require` I'd prefer to use ESModules with `import` statements:

```javascript
import htmlTagNames from 'html-tag-names'
import svgTagNames from 'svg-tag-names'
```